docs/assets/templates/services/template-minimum.dockstore.yml

```diff
@@ -0,0 +1,22 @@
+version: 1.2
+service:
+  subclass: <DOCKER_COMPOSE | KUBERNETES | HELM | SWARM | NOT_APPLICABLE>
+  name: <String>
+  authors:
+    - name: <String>
+      email: <String>
+  description: <String>
+  publish: <Boolean>
+  files: <String, do not use wildcards>
+  scripts:
+    start: <String>
+  environment:
+    <String-name-of-environment-variable>:
+      default: <String or Int>
```

Comment:
It's always treated as a string.

Reply:
Our larger template says...

`  #    - An optional default value (string or int), which will be used if the user does not specify in the input file
`

I don't want them to be inconsistent; which should I go for?

Reply:
String is correct. These literally get turned into Operating System environment variables, which have no type, AFAIK.